### DISCUSSION TITLE
What is This Box Called?

### DISCUSSION THREAD

**Discussion by SharpsterDraws** [keiwando/toonsquid#584] [General]
![IMG_9003](https://github.com/keiwando/toonsquid/assets/155850753/345c545d-53e9-4dc7-80ca-0ade3bf44b40)

I love this app…it’s so great… but what is this box and what creates this box around some layers and not others? I can’t draw outside of the box even though you can see the canvass behind it? 

Thank you in advance.

**Comment by DWrightAnimation** [comment]
That would be the canvas. You can only draw with raster brushes within the canvas, but vector brushes can go outside the canvas with no problem.

**Reply by SharpsterDraws** [reply]
And the canvas can move depending on its relation to the drawing I guess? Also I had edit pixel button on and that seems to affect the drawing and the new canvas.

**Reply by SharpsterDraws** [reply]
Okay, so I’m trying to understand the canvass and why it moves or acts differently with raster brushes and with vector brushes. I’m trying to understand why that happens?? 

Not complaining …. Again I love the app… my little brain just needs to know the theory or the rules behind it
